Docs/Signup_login.md

```diff
@@ -0,0 +1,79 @@
+# Signup and Login
+
+## Signup with Email
+From inside of Godot after you have set your configurations in the **Firebase.gd** script, you are able to call
+```
+Firebase.Auth.signup_with_email(email, password)
+```
+
+This will create the login_request_body variable and insert the correct data into it.
+
+```python
+var login_request_body = {
+    "email":"",
+    "password":"",
+    "returnSecureToken": true
+   }
+```
+
+From there the script will POST the data to the `signup_request_url` and add the user to the application
+
+## Login with Email
+From inside of Godot after you have set your configurations in the **Firebase.gd** script, you are able to call
+```
+Firebase.Auth.login_with_email(email, password)
+```
+
+This will create the login_request_body variable and insert the correct data into it.
+
+```python
+var login_request_body = {
+    "email":"",
+    "password":"",
+    "returnSecureToken": true
+   }
+```
+
+From there the script will POST the data to the `signin_request_url`, and wait for a response. The function `func _on_FirebaseAuth_request_completed(result, response_code, headers, body)` will take that response and parse it out for you.
+
+#### Unable to parse body
+If the script is unable to parse the body, it will print our an error to the console and 'return' out of the function
+
+
+#### Body has RESPONSE_SIGNIN, RESPONSE_SIGNUP
+If the response body has `RESPONSE_SIGNIN, RESPONSE_SIGNUP`, the user will be logged in and the refresh countdown will start
+
+#### Body has RESPONSE_USERDATA
+If the response body has `RESPONSE_USERDATA`, the script will emit a signal "userdata_recieved" with the userdata
```

Comment:
Slight typo, the signal is named: userdata_received (the ol' e/i switcheroo)

Reply:
Fixed it. I am the worst with typos